docs/apis/http/tus_upload.md

```diff
@@ -8,11 +8,10 @@ geekdocFilePath: tus_upload.md
 geekdocCollapseSection: true
 ---
 
-Infinite Scale supports the tus resumable-upload protocol.
-tus is a robust, modular, and open protocol designed to resume large file uploads reliably over HTTP.
+Infinite Scale supports the tus resumable-upload protocol, which is a robust, modular, and open protocol designed to resume large file uploads reliably over HTTP.
 In situations where file uploads might be interrupted due to network issues, browser crashes, or other unforeseen interruptions,
 tus ensures that uploads can be resumed from the point of failure without losing data.
-This documentation shows some basic examples, refer [tus official site](https://tus.io/protocols/resumable-upload) for more details.
+This documentation shows some basic examples, refer [Tus Official Site](https://tus.io/protocols/resumable-upload) for more details.
```

Comment:
```suggestion
This documentation shows some basic examples, refer to the [tus official site](https://tus.io/protocols/resumable-upload) for more details.
```

Reply:
No reason to capitalize "tus official site" here.

Reply:
gnargh! looks ugly....

Reply:
changed.